source/developer/fx-guidelines.rst

```diff
@@ -73,15 +73,29 @@ We’re using `BEM methodology <https://en.bem.info/method/>`_  and would advise
 
 -----
 
-Minimum Display Requirements
+Screen and Content Dimensions
 ****************************
 
-Desktop
--------
+Web/Desktop App
+-------------------
 
-- **Minimum screen resolution:** 1280 x 800
+The webapp supports two main modes of displaying the app: desktop view and mobile view. The app is primarily used in desktop view, but when the screen is 768 pixels or less wide, it switches to mobile view. The mobile view is also used in mobile browsers and the Classic App.
```

Comment:
What about tablet view?

Reply:
I don't think explicit tablet view actually exists any more. I tried resizing the window to trigger it, but I could never get the mobile send button to appear like it used to

Reply:
To add to that, there are some widths that do trigger some different CSS behaviours, but they aren't well defined. Some stuff changes at a width of 320px, 380px, 480px, 550px, 640px, 768px, 960px, 1020px, 1024px, 1140px, 1280px, 1440px, and 1800px. I assume most of those are just slight optimizations, but I'm not sure. Asaad would know more about that

Reply:
I think this sounds fine. We do not have a specific tablet view, we can say. 
`The webapp supports two main modes of displaying the app: desktop view and tablet/mobile view.`